packages/react/toggle/src/Toggle.test.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
```

Comment:
P.S. I'm not sure if the PR title follows the conventions. I looked into the commit history for reference. :)